libs/src/api/account.js

```diff
@@ -22,12 +22,23 @@ class Account extends Base {
    * @param {string} password
    */
   async login (email, password) {
+    const phases = {
+      WALLET_FOUND: 'WALLET_FOUND',
+      USER_FOUND: 'USER_FOUND'
+    }
+    let phase = ''
+
     if (!this.web3Manager.web3IsExternal()) {
       this.REQUIRES(Services.HEDGEHOG)
 
-      const ownerWallet = await this.hedgehog.login(email, password)
-      await this.web3Manager.setOwnerWallet(ownerWallet)
+      try {
```

Comment:
Should have documented this - I was thinking of phases as 'the thing we're currently trying to do'. So right now if we fail at the hedgehog loging step, `phase = ''` which ain't great. Would be good to have a phase that represents trying to log into hedgehog, and then a trying to get users, etc.